website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,101 @@
+
+## Exports vs. Saved Queries
+
+| Feature    | Exports  | Saved Queries  |
+|-----------|-----------|----------------|
+| **Availability**    | Available to dbt Cloud users on [Team or Enterprise](https://www.getdbt.com/pricing/) plans, on dbt versions 1.7 or higher.| Available in both dbt Core and dbt Cloud.     |
+| **Purpose**         | To materialize Saved Queries in your data platform, and expose metrics and dimensions as a view or table. | To define and manage common Semantic Layer queries in YAML, which includes metrics and dimensions.   |
+| **Usage**           | Automatically runs Saved Queries and materializes them within your data platform. Query metric defintions using SQL or integrate with a BI tool that doesn't have a native integration. Exports count towards [queried metrics](/docs/cloud/billing#what-counts-as-a-queried-metric) usage. <br /><br />**Example**: Creating a weekly aggregated table for active user metrics, automatically updated and stored in the data platform.  | Used for organizing and reusing common MetricFlow queries within dbt projects.<br /><br />**Example**: Group related metrics together for better organizations, and include commonly uses dimensions and filters. | For materializing query results in the data platform. |
+| **Integration**     | Must have the dbt Semantic Layer configured in your dbt project.<br /><br />Tightly integrated with the [MetricFlow Server](/docs/use-dbt-semantic-layer/sl-architecture#components) and dbt Cloud's job scheduler. | Integrated into dbt DAG and managed alongside other dbt nodes. |
+| **Configuration**   | Configured within dbt Cloud environment and job scheduler settings. | Defined in YAML format within dbt project files.   |
+
+## Prerequisites
+
+- Have a dbt Cloud account on a [Team or Enterprise](https://www.getdbt.com/pricing/) plan.
+- Be on dbt version 1.7 or higher.
+- Have a dbt Semantic Layer configured in your dbt project.
+- Have a dbt Cloud environment with a [job scheduler](/docs/deploy/job-scheduler) enabled.
+
+
+# Define Exports
+
+Exports are an additonal configuration added to a Saved Query. They define how this Saved Query should be materialized along with the schema and table name.
+You can define Exports in a YAML format in the as a key in the `saved_queries` configuration.
+
+```yaml
+saved_queries:
+  - name: my_query
+    description: null
+    query_params:
+      metrics:
+        - null
+      group_bys:
+        - TimeDimension()
+        - ...
+      where:
+        - null
+    export:
+      - name: my_export
+        config:
+          export_as: table # options: table, view
+          schema: my_schema # [optional - DEFAULT to deployment schema]
+          alias: some_table_name # [optional - DEFAULT to Export name]
+```
+## Running Exports
+
```

Comment:
@JStein77 is it `saved-query` or `saved_query`?

Reply:
saved-query!